_posts/2018-10-09-how-we-work-pair-programming.md

```diff
@@ -0,0 +1,110 @@
+# How We Work: Pair Programming
+
+by Jason Fry (@JasonMFry)
+
+![A photo of two people working at a pairing station][]
+use real photo, not this one.
+
+## Why We Pair
+
+1. Improves our Team
+    1. It forces us to become more personable, patient, and articulate
+    1. It makes junior engineers and new hires more productive faster
+    1. Spreads knowledge
+    1. Improves our cohesion as a team
+1. Improves our code
+    1. Two people make better design decisions than one person
+    1. Part of the Navigator's job is to spot opportunites for refactoring, so we're constantly
+       refactoring out codebase
+
+## How We Got Started Pairing
+
+We had an outside expert recommend that we begin pairing, and when we moved into a new building in
+October 2016 we took that opportunity to setup our area to make pairing easy (see
+[Logistics](#Logistics) for our setup). Thankfully our leadership is very supportive of experiments;
+in fact they encourage expirementation and celebrate well-intentioned failure.
+
+## Preferred Pairing Style
+
+There are different pairing methods, but we prefer Driver/Navigator over [Ping Pong][] because we
+don't do TDD and over [Strong Style][] because it feels foreign to us, though we may occasionally do
+it to shake things up. [Tour Guide][] Doesn't achieve our goals for knowledge transfer and code
+quality.
+
+We let each pair determine the specifics of each role. Below I've outlined how we _tend_ to define
+each role, with the caveat that we're still figuring it out ourselves.
+
+### Driver/Navigator
+
+The Driver is the one who operates the keyboard and mouse. They should think out loud and voice
+their expectations so that the Navigator knows what's going on.
+
+The Navigator tends to do most everything else:
+
+- Note opportunities for refactoring
+- Keep scope of story small
```

Comment:
I think most people will understand what we mean by story, but maybe saying "work" instead would be more direct? 
```suggestion
- Keep scope of work small
```